sql-plugin/pom.xml

```diff
@@ -49,45 +49,457 @@
             <artifactId>scalatest_${scala.binary.version}</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-          <groupId>org.apache.orc</groupId>
-          <artifactId>orc-core</artifactId>
-          <classifier>${orc.classifier}</classifier>
-          <exclusions>
-            <exclusion>
-              <groupId>org.slf4j</groupId>
-              <artifactId>slf4j-api</artifactId>
-            </exclusion>
-          </exclusions>
-        </dependency>
-        <dependency>
-          <groupId>org.apache.orc</groupId>
-          <artifactId>orc-mapreduce</artifactId>
-          <classifier>${orc.classifier}</classifier>
-          <exclusions>
-            <exclusion>
-              <groupId>com.google.code.findbugs</groupId>
-              <artifactId>jsr305</artifactId>
-            </exclusion>
-          </exclusions>
-        </dependency>
-        <dependency>
-          <groupId>org.apache.hive</groupId>
-          <artifactId>hive-storage-api</artifactId>
-          <exclusions>
-            <exclusion>
-              <groupId>org.slf4j</groupId>
-              <artifactId>slf4j-api</artifactId>
-            </exclusion>
-          </exclusions>
-        </dependency>
         <dependency>
           <groupId>com.google.protobuf</groupId>
           <artifactId>protobuf-java</artifactId>
         </dependency>
     </dependencies>
 
+    <dependencyManagement>
```

Comment:
dependencyManagement is useful for parent poms, but this isn't a parent pom.  It's also odd to have a dependencyManagement section with no versions listed, so I'm not sure what the point of this is.  All of the dependencies used in the normal dependency section below have versions as expected, so this seems unnecessary.

Reply:
Dependencies without version are used by different profiles, specified the exclusions,  so the dependencies in profiles can omit the exclusions.